recipes/onetbb/all/conanfile.py

```diff
@@ -56,10 +56,14 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def package_id(self):
-        if Version(self.version) < "2021.6.0":
+        if Version(self.version) < "2021.5.0":
             del self.info.options.tbbmalloc
+        if Version(self.version) < "2021.6.0":
             del self.info.options.tbbproxy
 
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.1 <4]")
+
```

Comment:
As already mentioned in another PR, CCI recipes don't add cmake to build requirements unless CMake > 3.15 is required. See https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-recipes-that-use-build-tools-like-cmake-that-have-packages-in-conan-center-do-not-use-it-as-a-build-require-by-default

Reply:
removed updating build dependencies.